addressvalidation/go.mod

```diff
@@ -0,0 +1,26 @@
+module cloud.google.com/go/addressvalidation
```

Comment:
Same as the other PR, I think we should group these under a maps module.

Reply:
Agreed, will do.